### ISSUE TITLE
Items not showing in various input modals

### ISSUE THREAD

**Issue by Chris-Petty** [openmsupply/open-msupply#896]
Inbound shipment:

![image](https://user-images.githubusercontent.com/7684221/205214036-fb10098e-b089-4357-a532-84c1da286a1d.png)

## Reproduction

Unsure. Fine if you add a new item. Occurs with lines from old requisitions that were made on mSupply Desktop and carried over to omSupply 

Console complains:

```
MUI: You have provided an out-of-range value `-1` for the select component.
Consider providing a value that matches one of the available options or ''.
The available values are `1`. 
SelectInput@http://localhost:3003/71687b9cc6a94ed75e14.js:54139:7
withEmotionCache/<@http://localhost:3003/71687b9cc6a94ed75e14.js:5145:66
div
withEmotionCache/<@http://localhost:3003/71687b9cc6a94ed75e14.js:5145:66
InputBase@http://localhost:3003/71687b9cc6a94ed75e14.js:43608:83
Input@http://localhost:3003/71687b9cc6a94ed75e14.js:42801:82
Select@http://localhost:3003/71687b9cc6a94ed75e14.js:53699:82
div
withEmotionCache/<@http://localhost:3003/71687b9cc6a94ed75e14.js:5145:66
FormControl@http://localhost:3003/71687b9cc6a94ed75e14.js:38082:82
withEmotionCache/<@http://localhost:3003/71687b9cc6a94ed75e14.js:5145:66
TextField@http://localhost:3003/71687b9cc6a94ed75e14.js:65869:83
../common/src/ui/components/inputs/Select/Select.tsx/Select<@http://localhost:3003/101a78e58ba95e103bec.js:15296:57
div
withEmotionCache/<@http://localhost:3003/71687b9cc6a94ed75e14.js:5145:66
Grid@http://localhost:3003/71687b9cc6a94ed75e14.js:39907:87
div
withEmotionCache/<@http://localhost:3003/71687b9cc6a94ed75e14.js:5145:66
Grid@http://localhost:3003/71687b9cc6a94ed75e14.js:39907:87
OutboundLineEditForm@http://localhost:3003/9fee50132dc0e638d229.js:9782:30
div
withEmotionCache/<@http://localhost:3003/71687b9cc6a94ed75e14.js:5145:66
Grid@http://localhost:3003/71687b9cc6a94ed75e14.js:39907:87
div
Transition@http://localhost:3003/71687b9cc6a94ed75e14.js:264252:30
Slide@http://localhost:3003/71687b9cc6a94ed75e14.js:55204:77
div
withEmotionCache/<@http://localhost:3003/71687b9cc6a94ed75e14.js:5145:66
DialogContent@http://localhost:3003/71687b9cc6a94ed75e14.js:35726:82
div
withEmotionCache/<@http://localhost:3003/71687b9cc6a94ed75e14.js:5145:66
Paper@http://localhost:3003/71687b9cc6a94ed75e14.js:51104:82
withEmotionCache/<@http://localhost:3003/71687b9cc6a94ed75e14.js:5145:66
div
withEmotionCache/<@http://localhost:3003/71687b9cc6a94ed75e14.js:5145:66
Transition@http://localhost:3003/71687b9cc6a94ed75e14.js:264252:30
Fade@http://localhost:3003/71687b9cc6a94ed75e14.js:37246:77
TrapFocus@http://localhost:3003/71687b9cc6a94ed75e14.js:13297:7
div
withEmotionCache/<@http://localhost:3003/71687b9cc6a94ed75e14.js:5145:66
Portal@http://localhost:3003/71687b9cc6a94ed75e14.js:11463:7
ModalUnstyled@http://localhost:3003/71687b9cc6a94ed75e14.js:10566:7
Modal@http://localhost:3003/71687b9cc6a94ed75e14.js:48866:82
withEmotionCache/<@http://localhost:3003/71687b9cc6a94ed75e14.js:5145:66
Dialog@http://localhost:3003/71687b9cc6a94ed75e14.js:35135:83
BasicModal@http://localhost:3003/101a78e58ba95e103bec.js:19946:59
ModalComponent@http://localhost:3003/101a78e58ba95e103bec.js:4298:28
OutboundLineEdit@http://localhost:3003/9fee50132dc0e638d229.js:9607:26
Provider@http://localhost:3003/71687b9cc6a94ed75e14.js:296826:20
Provider@http://localhost:3003/71687b9cc6a94ed75e14.js:296826:20
TableProvider@http://localhost:3003/101a78e58ba95e103bec.js:39323:44
Suspense
DetailView@http://localhost:3003/9fee50132dc0e638d229.js:8834:47
Routes@http://localhost:3003/71687b9cc6a94ed75e14.js:260552:7
InvoiceService
DistributionRouter@http://localhost:3003/101a78e58ba95e103bec.js:51086:102
Routes@http://localhost:3003/71687b9cc6a94ed75e14.js:260552:7
div
withEmotionCache/<@http://localhost:3003/71687b9cc6a94ed75e14.js:5145:66
Box@http://localhost:3003/71687b9cc6a94ed75e14.js:75312:72
div
withEmotionCache/<@http://localhost:3003/71687b9cc6a94ed75e14.js:5145:66
Box@http://localhost:3003/71687b9cc6a94ed75e14.js:75312:72
SnackbarProvider@http://localhost:3003/71687b9cc6a94ed75e14.js:216392:24
KBarProvider@http://localhost:3003/71687b9cc6a94ed75e14.js:201617:48
CommandK@http://localhost:3003/101a78e58ba95e103bec.js:44326:18
RequireAuthentication@http://localhost:3003/101a78e58ba95e103bec.js:50014:31
Site@http://localhost:3003/101a78e58ba95e103bec.js:44832:93
Routes@http://localhost:3003/71687b9cc6a94ed75e14.js:260552:7
div
withEmotionCache/<@http://localhost:3003/71687b9cc6a94ed75e14.js:5145:66
Box@http://localhost:3003/71687b9cc6a94ed75e14.js:75312:72
Viewport@http://localhost:3003/101a78e58ba95e103bec.js:50766:80
Router@http://localhost:3003/71687b9cc6a94ed75e14.js:260490:7
BrowserRouter@http://localhost:3003/71687b9cc6a94ed75e14.js:259327:7
AlertModalProvider@http://localhost:3003/101a78e58ba95e103bec.js:19635:28
ConfirmationModalProvider@http://localhost:3003/101a78e58ba95e103bec.js:20431:35
InnerThemeProvider@http://localhost:3003/71687b9cc6a94ed75e14.js:74543:70
ThemeProvider@http://localhost:3003/71687b9cc6a94ed75e14.js:74074:7
ThemeProvider@http://localhost:3003/71687b9cc6a94ed75e14.js:74565:7
div
RTLProvider@http://localhost:3003/101a78e58ba95e103bec.js:9136:101
LocalizationProvider@http://localhost:3003/71687b9cc6a94ed75e14.js:85017:7
ThemeProvider@http://localhost:3003/101a78e58ba95e103bec.js:9235:23
AuthProvider@http://localhost:3003/101a78e58ba95e103bec.js:978:22
GqlProvider@http://localhost:3003/101a78e58ba95e103bec.js:187:21
QueryClientProvider@http://localhost:3003/71687b9cc6a94ed75e14.js:257637:16
ErrorBoundary@http://localhost:3003/101a78e58ba95e103bec.js:12675:9
Suspense
I18nextProvider@http://localhost:3003/71687b9cc6a94ed75e14.js:250869:14
IntlProvider@http://localhost:3003/101a78e58ba95e103bec.js:7522:22
Suspense
Host
App 2 [react_devtools_backend.js:4012:25](moz-extension://fa30151d-f681-4c0d-a450-07dba6b6b7ba/build/react_devtools_backend.js)

```

internal order: 

![image](https://user-images.githubusercontent.com/7684221/205214065-b1ed851a-0e13-48a1-8642-7419d63a9e79.png)

No console complaints.

## Reproduction (internal order)

1. New order
2. Add a line
3. Reopen the line... NOTE: it works when you do this
4. Add from master list
5. Open a line... No item name displayed

**Comment by mark-prins** [comment]
Possibly #961 is a duplicate
Possibly - this is due to limit on the number of items returned, so could be resolved with [this change](https://github.com/openmsupply/open-msupply/pull/897/files#diff-4b04a01d3728d06a4888dbae1ba669cd886ead68362394d54768c4a27716407f)

**Comment by Chris-Petty** [comment]
@mark-prins Sorry I probably won't be able to - I am focusing on sync issues for this last week!

**Comment by mark-prins** [comment]
Can confirm that this is resolved with the fix in #897 et al
